pm/ticket.go

```diff
@@ -0,0 +1,109 @@
+package pm
+
+import (
+	"math/big"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+)
+
+// Constants for byte sizes of Solidity types
+const (
+	addressSize = 20
+	uint256Size = 32
+	bytes32Size = 32
+)
+
+// Broker is an interface which serves as an abstraction over an on-chain
+// smart contract that handles the administrative tasks in a probabilistic micropayment protocol
+// including processing deposits and pay outs
+type Broker interface {
```

Comment:
What are the pros/cons of having the `Broker` interface in the `ticket.go` file, versus creating a new file like `broker.go` ?

Reply:
I didn't have a strong reason for keeping the `Broker` interface in `pm/ticket.go`. Since there is not an concrete implementation for this interface yet, I didn't bother putting it into it's own file. I can move this interface into it's own file thought - it seems cleaner and will probably be helpful later on for the concrete implementation